archive/archive_bags/src/archive_bags.py

```diff
@@ -0,0 +1,39 @@
+# -*- encoding: utf-8 -*-
+"""
+Returns a manifest file requested by id in VHS where dynamodb stores the id to S3 location mapping.
+"""
+
+import daiquiri
+import json
+import os
+import boto3
+from wellcome_aws_utils.lambda_utils import log_on_error
+
+TABLE_NAME = 'vhs-archive-manifests'
+BUCKET_NAME = 'wellcomecollection-vhs-archive-manifests'
+
+daiquiri.setup(level=os.environ.get('LOG_LEVEL', 'INFO'))
+logger = daiquiri.getLogger()
+
+
+@log_on_error
+def main(event, _, dynamodb_client=None, s3_client=None):
+    dynamodb_client = dynamodb_client or boto3.client('dynamodb')
+    s3_client = s3_client or boto3.client('s3')
```

Comment:
I noticed now that `dynamo_resource` is an option, and we should move to that in another PR.